ci-operator/config/openshift/ci-tools/openshift-ci-tools-master.yaml

```
@@ -507,7 +493,7 @@ tests:
       - mount_path: /tmp/hive
         name: hive-hive-credentials
         namespace: test-credentials
-      from: e2e-bin
+      from: bin
```

test-bin has the binaries with `-race`:
```suggestion
      from: test-bin
```